daemon/state.go

```diff
@@ -246,25 +246,21 @@ func readEPsFromDirNames(basePath string, eptsDirNames []string) []*endpoint.End
 // syncLabels syncs the labels from the labels' database for the given endpoint.
 // To be used with endpoint.Mutex locked.
 func (d *Daemon) syncLabels(ep *endpoint.Endpoint) error {
-	if ep.SecurityIdentity == nil {
-		return fmt.Errorf("Endpoint doesn't have a security label.")
-	}
-
 	// Filter the restored labels with the new daemon's filter
-	l, _ := labels.FilterLabels(ep.SecurityIdentity.Labels)
-	ep.SecurityIdentity.Labels = l
-
-	identity, _, err := identityPkg.AllocateIdentity(ep.SecurityIdentity.Labels)
+	l, _ := labels.FilterLabels(ep.OpLabels.AllLabels())
```

Comment:
This also includes disabled labels. That doesn't seem right. I think you want to use `e.OpLabels.IdentityLabels`

Reply:
oh, awesome, that's exactly what I want.